snippets/cdc-acm-console/Kconfig

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2023-2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config SHELL_BACKEND_SERIAL_CHECK_DTR
```

Comment:
This is not actually a `Kconfig` file, it's actually a `Kconfig.defconfig` file, `Kconfig` files add settings and `Kconfig.defconfig` files set defaults. I do not think snippets should be adding `Kconfig` because all that's going to do is cause confusion or require someone to do `-DCONFIG_x=y` on the command line when using snippets and the whole point in snippets is to not have to do `-DCONFIG_x=y`